TTS/trainer.py

```diff
@@ -764,11 +765,13 @@ def test_run(self) -> None:
         """Run test and log the results. Test run must be defined by the model.
         Model must return figures and audios to be logged by the Tensorboard."""
         if hasattr(self.model, "test_run"):
-            if hasattr(self.eval_loader, "load_test_samples"):
-                samples = self.eval_loader.load_test_samples(1)
-                figures, audios = self.model.test_run(samples)
+            if isinstance(self.model, Wavegrad):
```

Comment:
Here I had to set an exception for Wavegrad, it seems that the problem comes from the inference function that eats all the memory available. Not linked to this PR (and I don't know how to fix it)

Reply:
Like I said, I wouldn't know how to fix the memory issue because I never worked with wavegrad. Are you suggesting that I should remove the test_run function from wavegrad ? I don't think it's that function that is broken but instead inference. I don't know at what level it would be better to disable wavegrad's test_run.

Reply:
Did you try training and caused OOM  or just the CI tests?

Reply:
OOM with both CUDA 24Gb of VRAM and CPU only with 32Gb + 16GB SWAP

Reply:
It is when you train the model right not CI?  Do you have any script to replicate the issue ?

Reply:
No during the execution of the tests, maybe you can try to replicate by pulling this branch. The crash wasn't occurring before because run_test was never called.

Reply:
I can check Wavegrad next week. For now, we can overlook the issue it causes. So we don't need to handle it specially.

Reply:
Ok so all good then?